src/MoBi.Presentation/Tasks/Edit/EditTaskFor.cs

```diff
@@ -98,7 +98,7 @@ private IMoBiCommand rename(T objectBase, IEnumerable<string> forbiddenNames, IB
             Description = AppConstants.Commands.RenameDescription(objectBase, newName)
          };
 
-         if (CheckUsagesFor(newName, objectBase.Name, objectBase, commandCollector, buildingBlock?.Module))
+         if ((objectBase is IBuildingBlock) || CheckUsagesFor(newName, objectBase.Name, objectBase, commandCollector, buildingBlock?.Module))
```

Comment:
I would just explain in a comment why this check is here. You could also return true from `CheckUsagesFor` with this type check rather than in this if statement.